src/pages/news/2021-04-16-codecov-security-update.en.md

```diff
@@ -0,0 +1,149 @@
+---
+path: "/en/news/2021-04-16-codecov-security-update/"
+date: "2021-04-16"
+title: "Codecov security update: Open Zaak not affected"
+lang: en
+template: default
+---
+
+## Codecov security update: Open Zaak is not affected
+
+On April 15th, we were informed by Codecov that they had a security incident that
+possibly affected Open Zaak repositories. We have checked our repositories and verified
+that everything is as expected.
+
+Unless you are using the `latest` image tag of Open Zaak
+or Open Notificaties Docker Hub images, **no action is required**. If you _do_ use the `latest` 
+tag in any of those images, our advise is to pull these images again. There is no indication 
```

Comment:
I might write this as:

If you _do_ use the `latest` tag in any of those images, _and_ those images are ole enough to no longer have build logs, then our advice is to pull these images again.

However, I'm also fine with it as-is.

Reply:
oh, good catch! I like your version better, will change it :)